### ISSUE TITLE
HacktoberFest2k20

### ISSUE THREAD

**Issue by apoorvdwi** [apoorvdwi/HacktoberFest2k20#2]
# Hello EveryOne

Want to contribute to hacktoberfest-2020 the month long celebration of open source ?
Are you beginner or just starting to contribute to open source ?

This repository contains many data structure and algorithm problems .
Modify the existing one or add your own algorithms.
P.S.: Please make sure the file name itself is self-explanatory about the Algorithm or make a folder along containing your script and a README file explaining the necessary details.

**Comment by PabloCorbCon** [comment]
I have [this](https://github.com/PabloCorbCon/codewars-examples) repository about my competitive programming examples in codewars, I started using Codewars a week ago or so since I have been using Coderrank for the past two years but I decided to change. Do you think the examples I have in that repository are similar enough to what you want for this one? If you tell me that I can do pulls right now.

**Comment by apoorvdwi** [comment]
sure make a seperate folder and make the PR's .

**Comment by saiharsha-22** [comment]
@apoorvdwi  Hi, I would like to contribute. can I add Binary Search algorithms in Java programming?

**Comment by saiharsha-22** [comment]
I will arrange it in a proper folder and add it.

**Comment by apoorvdwi** [comment]
@saiharsha-22 Yeah offcourse there is a java folder already in algorithms . You can put that in there .

**Comment by saiharsha-22** [comment]
@apoorvdwi Thank you I will start doing the work.

**Comment by saiharsha-22** [comment]
@apoorvdwi  I have sent a pull request can you please check it once.

![Screenshot (391)](https://user-images.githubusercontent.com/61947484/94799813-59065e00-0401-11eb-8f1f-43cc470d95de.png)

**Comment by Mayankjha997** [comment]
hey!! can i contribute in this repo i can add algorithm of rock paper scissor game in c language

**Comment by NidhiNivedita6** [comment]
I can add algorithm of NEWTON- RAPHSON METHOD in C language.

**Comment by Aaditya1978** [comment]
@apoorvdwi can I add the algorithm for printing Pascal's Triangle in C. I will add it in the C folder in Algorithms Folder.

**Comment by apoorvdwi** [comment]
@Aaditya1978 Go ahead .

**Comment by Aaditya1978** [comment]
@apoorvdwi  Please Merge my PR #39

**Comment by abhaydhiman** [comment]
@apoorvdwi I want to add a machine learning task that is of stock price prediction using `LSTM` model. Can I add that?

**Comment by apoorvdwi** [comment]
@abhaydhiman  
sure add the file in models folder in machine learning

**Comment by abhaydhiman** [comment]
Ok, I am on it!!

**Comment by Ayush-Malik** [comment]
I have a special diamond pattern printing pattern problem in both (Python and c) language with 
only :
1 Sinlge Loop 
2 Single if - else if ladder
3 Only 3 variables in whole program

Can i make a PR on it

**Comment by abhaydhiman** [comment]
@apoorvdwi I want to add a `Linked list` program in `python3` that consists of:-
* Insertion of the node
* Deletion of the node
* Node finder based on the index value.

Can I make a PR on it?

**Comment by Resolution-1** [comment]
i would like to work on that issue

**Comment by TILhub** [comment]
Hey can you please merge my pull requests
